infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -315,6 +314,16 @@ public void handleTokenQuery(CorfuPayloadMsg<TokenRequest> msg,
     }
 
     /**
+     * Service an incoming request to mark the server as NOT READY.
+     */
```

Comment:
My opinion matches Dahlia's: each sequencer instance should be able to figure this out easily.  The combination of adding the SET_NOT_READY message and also relying on "somebody else"(*) to send them to all sequencer instances (and handling failures delivering those messages) is new complexity that should be avoided, if feasible.  If adding SET_NOT_READY is indeed needed, then an explanation & example of why would be very useful.

(*) My quotation marks, not actually quoting Zeeshan.